Widgets_CDAT/DictWidget/test_keyValue.py

```diff
@@ -0,0 +1,76 @@
+from PySide.QtCore import *
+from PySide.QtGui import *
+from collections import OrderedDict
+from dict import DictEditor
```

Comment:
So, naming the module dict is a "bad practice", because `dict` is a built-in type (`{}` == `dict()`). You should probably rename the module.